client/consensus/slots/src/lib.rs

```diff
@@ -158,6 +161,12 @@ pub trait SimpleSlotWorker<B: BlockT> {
 	/// Whether to force authoring if offline.
 	fn force_authoring(&self) -> bool;
 
+	/// The strategy for backing off block authorship is not set by default, since it usually
+	/// requires state that we do don't have here.
```

Comment:
This is no real documentation on what this function is doing/should do :P

```suggestion
	/// Returns whether the block production should back off.
	/// 
	/// By default this function always returns `false`.
	///
	/// An example strategy that back offs if the finalized head is lagging too much behind the tip is
	/// implemented by [`SimpleBackoffAuthoringBlocksStrategy`].
```